thirdparty/miniupnpc/src/miniupnpcstrings.h

```diff
@@ -1,9 +1,7 @@
 #ifndef MINIUPNPCSTRINGS_H_INCLUDED
 #define MINIUPNPCSTRINGS_H_INCLUDED
 
-#include "core/version.h"
-
-#define OS_STRING VERSION_NAME "/1.0"
+#define OS_STRING "Godot Engine/1.0"
```

Comment:
CC @mhilbrunner @Faless 

The previous code was a good idea in general, and allows tweaking this in Godot forks with a different name. But it was the only thirdparty code reaching into `core/version.h`, and actually `core/version_generated.gen.h`, which is recreated for each commit (includes Git hash) so that means this header and the miniupnpc files using it would always get recompiled. Doesn't seem worth it.

Reply:
BTW if we want to keep it dynamic, we could instead parse `version.py` from `modules/upnp/SCsub` and pass `OS_STRING` on the command line. But I don't think it's worth the trouble, at least until someone asks for it.

Reply:
Well it's not as bad as I mentioned actually, `version_generated.gen.h` doesn't contain the commit hash, this one is split to `version_hash.gen.h` which is not included by default in `version.h`.

`version_generated.gen.h` shouldn't change too often:
```
/* THIS FILE IS GENERATED DO NOT EDIT */
#ifndef VERSION_GENERATED_GEN_H
#define VERSION_GENERATED_GEN_H
#define VERSION_SHORT_NAME "godot"
#define VERSION_NAME "Godot Engine"
#define VERSION_MAJOR 4
#define VERSION_MINOR 3
#define VERSION_PATCH 0
#define VERSION_STATUS "dev"
#define VERSION_BUILD "custom_build"
#define VERSION_MODULE_CONFIG ""
#define VERSION_WEBSITE "https://godotengine.org"
#define VERSION_DOCS_BRANCH "latest"
#define VERSION_DOCS_URL "https://docs.godotengine.org/en/" VERSION_DOCS_BRANCH
#endif // VERSION_GENERATED_GEN_H
```

So I could restore this if we want, though I still feel it's cleaner not to have thirdparty code reach out to our `core` if we can avoid it.

Reply:
No I agree, changing this seems fine.